shuffle.js

```diff
@@ -0,0 +1,43 @@
+const all = document.querySelector('#all');
+const web_design = document.querySelector('#web_design');
+const graphic_design = document.querySelector('#graphic_design');
+const artwork = document.querySelector('#artwork');
+const shuffle_container = document.querySelector('#shuffle');
+
+function shuffle(container) {
+    for (var i = container.children.length; i >= 0; i--) {
+        container.appendChild(container.children[Math.random() * i | 0]);
+    }
+}
+
+all.addEventListener('click', () => {
+    web_design.classList.remove('active_item');
+    graphic_design.classList.remove('active_item');
+    artwork.classList.remove('active_item');
+    all.classList.add('active_item');
+    shuffle(shuffle_container);
```

Comment:
you are repeating almost identical steps in all callbacks, so probably it is possible to make a function that handles all this class toggling stuff. and i guess it is enough to have one event listener on body, where you can play with event.target.

Reply:
it is fine for the sake of the current task, but one you will study dom manipulation with js such solutions will not be acceptable